app/models/link_graph/node.rb

```diff
@@ -68,4 +80,20 @@ def to_h
       links: children,
     }
   end
+
+  def might_have_own_links?
```

Comment:
well it's because the field can be true|false|null. When it is true we have_own_links when false we don't have them, and when it's null it's a maybe.